signature.py

```diff
@@ -1,14 +1,7 @@
-from eth_keys.datatypes import (
-    Signature,
-)
-
-from eth_utils import (
-    to_checksum_address,
-)
-
-from eth_keys.utils.address import (
-    public_key_bytes_to_address,
-)
+from eth_keys.datatypes import Signature
+from eth_keys.exceptions import ValidationError
```

Comment:
This seems unused.

Reply:
ValidationError is used from app.py:
```
from signature import recover_public_address, ValidationError as SignatureValidationError
```

Reply:
Oh didn't notice that - that's a bit confusing, but kinda makes sence, that we're using `eth_keys` package only in `signature.py`.